bitnami/grafana-alloy/values.yaml

```diff
@@ -0,0 +1,933 @@
+  ## More information: https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_resources.tpl#L15
+  ##
+  resourcesPreset: "nano"
+  ## @param alloy.resources Set Grafana Alloy container requests and limits for different resources like CPU or memory (essential for production workloads)
+  ## Example:
+  ## resources:
+  ##   requests:
+  ##     cpu: 2
+  ##     memory: 512Mi
+  ##   limits:
+  ##     cpu: 3
+  ##     memory: 1024Mi
+  ##
+  resources: {}
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ## @param alloy.containerSecurityContext.enabled Enabled Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.seLinuxOptions [object,nullable] Set SELinux options in Grafana Alloy container
+  ## @param alloy.containerSecurityContext.runAsUser Set runAsUser in Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.runAsGroup Group ID for Grafana Alloy's containers
+  ## @param alloy.containerSecurityContext.runAsNonRoot Set runAsNonRoot in Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.readOnlyRootFilesystem Set readOnlyRootFilesystem in Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.privileged Set privileged in Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.allowPrivilegeEscalation Set allowPrivilegeEscalation in Grafana Alloy container's Security Context
+  ## @param alloy.containerSecurityContext.capabilities.drop List of capabilities to be dropped in Grafana Alloy container
+  ## @param alloy.containerSecurityContext.seccompProfile.type Set seccomp profile in Grafana Alloy container
+  ##
+  containerSecurityContext:
+    enabled: true
+    seLinuxOptions: {}
+    runAsUser: 1001
+    runAsGroup: 1001
+    runAsNonRoot: true
+    readOnlyRootFilesystem: true
+    privileged: false
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop: ["ALL"]
+    seccompProfile:
+      type: "RuntimeDefault"
+  ## @param alloy.lifecycleHooks for Grafana Alloy containers to automate configuration before or after startup
+  ##
+  lifecycleHooks: {}
+  ## @param alloy.extraEnvVars Array with extra environment variables to add to Grafana Alloy containers
+  ## e.g:
+  ## extraEnvVars:
+  ##   - name: FOO
+  ##     value: "bar"
+  ##
+  extraEnvVars: []
+  ## @param alloy.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for Grafana Alloy containers
+  ##
+  extraEnvVarsCM: ""
+  ## @param alloy.extraEnvVarsSecret Name of existing Secret containing extra env vars for Grafana Alloy containers
+  ##
+  extraEnvVarsSecret: ""
+  ## @param alloy.extraVolumes Optionally specify extra list of additional volumes for the Grafana Alloy pods
+  ##
+  extraVolumes: []
```

Comment:
`extraVolumes` is a per pod feature (not per container), then it should be `extraVolumes` given all the `alloy.*` parameters you have only apply to the Grafana Alloy container.